docs/migration/index.md

```diff
@@ -42,7 +42,7 @@ If you use the [`--configPlugin`](../command-line-interface/index.md#configplugi
 
 Some options now have different default values. If you think you experience any issues, try adding the following to your configuration:
 
-```
+```js
```

Comment:
Unfortunately, this is not really JavaScript, at least Syntax highlighting would not get this correctly (it is a block statement with lots of labels and comma expressions).

Reply:
Oh, I assumed as a valid JS without giving much thought because the syntax highlight colored the text. But you're correct. This isn't a valid JS. I'll add the parentheses.